content/en/docs/tutorials/kubernetes-basics/explore/explore-intro.html

```diff
@@ -164,15 +164,15 @@ <h3>Show the app in the terminal</h3>
                 <h3>View the container logs</h3>
                 <p>Anything that the application would normally send to standard output becomes logs for the container within the Pod. We can retrieve these logs using the <code>kubectl logs</code> command:</p>
                 <p><code><b>kubectl logs "$POD_NAME"</b></code></p>
-                <p><em><strong>Note:</strong> We don't need to specify the container name, because we only have one container inside the pod.</em></p>
+                <p><em><strong>Note:</strong> We don't need to specify the container name, because we only have one container inside the Pod.</em></p>
            </div>
         </div>
 
         <div class="row">
             <div class="col-md-12">
                 <h3>Executing command on the container</h3>
                 <p>We can execute commands directly on the container once the Pod is up and running.
-                For this, we use the <code>exec</code> subcommand and use the name of the Pod as a parameter. Let’s list the environment variables:</p>
+                For this, we use the <code>exec</code> subcommand and use the name of the Pod as a parameter. Let’s list the environment variables in our container:</p>
```

Comment:
Actually, we [avoid using “we”](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-using-we) and “our”.

Reply:
Still working on this?